cli/src/cmd/forge/script/broadcast.rs

```diff
@@ -286,65 +325,150 @@ impl ScriptArgs {
         Ok(())
     }
 
+    /// Certain features are disabled for multi chain deployments, and if tried, will return
+    /// error.
+    fn check_multi_chain_constraints(
+        &self,
+        total_rpcs: usize,
+        libraries: &Libraries,
+    ) -> eyre::Result<()> {
+        if total_rpcs > 1 {
+            eprintln!(
+                "{}",
+                Paint::yellow(
+                    "Multi chain deployment is still under development. Use with caution."
+                )
+            );
+            if !libraries.libs.is_empty() {
+                eyre::bail!(
+                    "Multi chain deployment does not support library linking at the moment."
+                )
+            }
+        }
+        Ok(())
+    }
+
     /// Modify each transaction according to the specific chain requirements (transaction type
     /// and/or gas calculations).
```

Comment:
let's add what those are

Reply:
ended up renaming it (`bundle_transactions`) and changing the doc